package.json

```diff
@@ -2,7 +2,7 @@
   "name": "nextjs-commerce",
```

Comment:
Could you also revert these to match with `commerce:master` and create a seperate branch for the package updates?